pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -598,6 +598,16 @@ def _process_sites_weekly_ts(coldata, regions_how, region_ids, meta_glob):
         ),
     }
 
+    # try to merge repw_res together into one dataarray
+    repw_res_v2 = xr.concat(
+        [
+            _create_diurnal_weekly_data_object(coldata, "seasonal")["rep_week"],
+            _create_diurnal_weekly_data_object(coldata, "yearly")["rep_week"].expand_dims(
+                "period", axis=0
+            ),
+        ],
+        dim="period",
+    )
```

Comment:
this dataset looks unused

Reply:
that is correct (wondering what the purpose was). But since this is diurnal cycle this likely was WIP when Jonas left. removed